### DISCUSSION TITLE
I failed connecting my Android phone using the Nebula app

### DISCUSSION THREAD

**Discussion by zoechi** [slackhq/nebula#1296] [General]
I tried to copy/paste the content of the ca.crt into the `Load CA` field, but when I click `Load CA` it silently discards the content. When I select a file, it discards it as well (nothing happens).

I wanted to try QR-code but I don't understand the parameters the command `nebula-cert print` expects.

I tried

```sh
nebula-cert print -path keys/nebula/ca.crt -out-qr
```

but `-out-q` expects another string parameter and I haven't found any hint about what value that should be.

I'm completely new to Nebula and just try to get started.

**Comment by johnmaguire** [comment]
Sorry about the issues loading the CA. `-out-qr` should be set to the filename you want the barcode PNG to be saved to.

**Reply by zoechi** [reply]
Thanks for looking into it.
I just stumbled over the generated QR file that was generated when I added some random string yesterday while experimenting.
~The scanning works but the scanned QR code is silently ignored exactly as the pasted text or the selected file.~

I just checked again. It's shown after clicking the button.
Some indication that something happened and that it was successful would be very helpful.
